tests/transforms/test_gaussian_blur.py

```diff
@@ -13,6 +14,13 @@ def test_on_pil_image(self) -> None:
                 sample = Image.new("RGB", (w, h))
                 gaussian_blur(sample)
 
+    def test_on_tensor(self) -> None:
+        for w in range(1, 100):
+            for h in range(1, 100):
```

Comment:
Doesn't have to be fixed now but the loop with two ranges runs 10'000 tests which seems excessive. Same for the test above. 

```suggestion
        for w in [1, 10, 100]:
            for h in [1, 10, 100]:
```